core/api/src/servers/event-handlers/bria.ts

```diff
@@ -46,8 +61,17 @@ export const briaEventHandler = async (event: BriaEvent): Promise<true | DomainE
         payoutInfo: event.augmentation.payoutInfo,
       })
 
+    case BriaPayloadType.PayoutCommitted:
+      return true
+
     case BriaPayloadType.PayoutCancelled:
-      return payoutCancelledEventHandler({ event: event.payload })
+      if (event.augmentation.payoutInfo === undefined) {
+        return new EventAugmentationMissingError()
```

Comment:
we cant block event processor because of this... stablesats didnt use externalId, also we should stop using === undefined (payout info can be null)

![image](https://github.com/GaloyMoney/galoy/assets/2079600/0ccf504f-77c9-4aca-9595-1a69971a5252)

Reply:
It should never be null when passing augment option

Reply:
Also don't understand relevance of `stablesats didnt use externalId`? Since rollout on Monday it is actually using `externalId` - but what does that have to do with anything?

Reply:
> we cant block event processor because of this... stablesats didnt use externalId, also we should stop using === undefined (payout info can be null)

payoutInfo can't be null in this scenario because we always pass the `augmentation`. In the instances where an externalId isn't explicitly passed to stablesats (before Monday) a random uuid is internally assigned. Can see the difference in the two clusters below.

Traces: https://ui.honeycomb.io/galoy/datasets/galoy-bbw/result/qEnUKXWF6pf

Reply:
Its not random - its the internal id https://github.com/GaloyMoney/bria/blob/3e25e4a3ebd685967ae422b234a3a6aa895566e8/src/payout/entity.rs#L89-L93

Reply:
> Also don't understand relevance of `stablesats didnt use externalId`? Since rollout on Monday it is actually using `externalId` - but what does that have to do with anything?

we need to reprocess txs